lib/signet/oauth_2/client.rb

```diff
@@ -189,6 +192,7 @@ def update!(options={})
         self.signing_key = options[:signing_key] if options.has_key?(:signing_key)
         self.extension_parameters = options[:extension_parameters] || {}
         self.additional_parameters = options[:additional_parameters] || {}
+        self.access_type = options.fetch(:access_type) { :offline }
```

Comment:
can you update to match the format of the previous line `option[:key] || default_value`

Reply:
this line behaves differently

it allows to pass `access_type: nil` which will prevent signet from adding `access_type=offline` in payload.
but it shouldn't change functionality if you dont' provide `access_type` key because it will break existing applications that works with google OAuth2